Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PipelineBrush/PipelineBrushXaml.bind

```diff
@@ -20,10 +20,10 @@
               Grid.Column="2"
               Height="400">
         <Border.Background>
-          <brushes:PipelineBrush Source="{effects:BackdropSource BackgroundSource=Backdrop}">
+          <brushes:PipelineBrush Source="{effects:BackdropSource}">
```

Comment:
What happens in a PipelineBrush if you don't set a `Source`?

Reply:
You should get a `NullReferenceException` when the brush is initialized.

Reply:
That sounds good, or I was thinking we could initialize a default Backdrop brush (if that doesn't add extra overhead)?

Reply:
That could be done by just changing:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/0db6b4ca10ae586e882fb5043b5ba13ee4deaf69/Microsoft.Toolkit.Uwp.UI.Media/Brushes/PipelineBrush.cs#L33

With:

```csharp
PipelineBuilder builder = Source ?? PipelineBuilder.FromBackdrop();
```

Which wouldn't add overhead at all as it'd be skipped most of the times.

I'm just not sure that'd be the best desired behavior for devs, like, personally I'd just prefer a clear exception run once, so that I would see it while debugging and immediately fix it, rather than having a default behavior which might not be the one I was looking for, and without a clear indication of why that's happening 🤔

What do you think?